src/DbPostgreSQLAdapter/FieldValue.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "DbAdapterInterface/IFieldValue.h"
+#include "DbAdapterInterface/IBinaryValue.h"
+
+namespace systelab::db {
+	class IField;
+}
+
+namespace systelab::db::postgresql {
+
+	class FieldValue : public IFieldValue
+	{
+	public:
+
+		FieldValue(const IField&);
+		FieldValue(const IField&, bool);
+		FieldValue(const IField&, int);
+		FieldValue(const IField&, double);
+		FieldValue(const IField&, const std::string&);
```

Comment:
Similar comment to AdapterInterface for StubFieldValue. We should prevent unexpected conversion from `const char*` to bool getting the bool constructor.

Reply:
Good catch, I'll change it